test/e2e/kubevirt.go

```diff
@@ -630,6 +638,23 @@ iperf3 -t 0 -c %[1]s -p %[2]d --logfile %[3]s &
 			)
 		}
 
+		liveMigrateFailed = func(vmi *kubevirtv1.VirtualMachineInstance) {
```

Comment:
GinkgoHelper?

Reply:
What do you mean ?

Reply:
Add call to GinkgoHelper() within this function as it is being done in all other functions that use gomega matchers.

Reply:
Ahh right, I spaced out, on it.